CHANGELOG.md

```diff
@@ -63,6 +63,10 @@ For older changes see the [archived Singularity change log](https://github.com/a
 - The `instance start` command now accepts an optional `--app <name>` argument which
   invokes start script within the `%appstart <name>` section in the definition file.
   The `instance stop` command still only requires the instance name.
+- Templating support for definition file. Users can define variables in definition
+  file via a matching pair of double curly brackets `{{ variable }}`, defined variables
+  will be replaced later either by `variable=value` entry defined in `%default` section
+  in definition file or through new cli options `--build-arg` and `--build-arg-file`.
```

Comment:
A little cleanup of the English:
```
- Templating support for definition files. Users can now define variables in definition
  files via a matching pair of double curly brackets. Variables of the form `{{ variable }}` 
  will be replaced by a value defined either by a `variable=value` entry in the `%default`
  section of the definition file or through new build options `--build-arg` or `--build-arg-file`.
```

Reply:
resolved and squashed in this commit
https://github.com/apptainer/apptainer/pull/1344/commits/377b9e8a8bd299caef150607c1463085acd871db